src/tasks/S3Upload/TaskOperations.ts

```diff
@@ -171,11 +172,11 @@ export class TaskOperations {
             if (!stats.isDirectory()) {
                 const fileBuffer = fs.createReadStream(matchedFile)
                 try {
-                    let contentType: string | undefined
+                    let contentType: string | false | undefined
                     if (this.taskParameters.contentType) {
                         contentType = this.taskParameters.contentType
                     } else {
-                        contentType = knownMimeTypes.get(path.extname(matchedFile))
+                        contentType = lookup(path.extname(matchedFile))
                         if (!contentType) {
```

Comment:
Is there a test that covers when lookup returns false?

Reply:
I can try and add one, the problem is that we have to use the project's file tree. The dotfiles in the home dir might work.

Reply:
Add something like ".thiswillneverresolveamimetype" that will never work